app/controllers/hub/efile_submissions_controller.rb

```diff
@@ -22,7 +22,7 @@ def show
       # Eager-load tax returns with submissions and data we may need to render
       @tax_returns = client.tax_returns.includes(:efile_submissions, efile_submissions: :fraud_score).where.not(tax_returns: {efile_submissions: nil}).load
       @fraud_indicators = Fraud::Indicator.unscoped
-      redirect_to hub_client_path(id: @client.id) and return unless @tax_returns.present?
+      redirect_to hub_client_path(id: @client.id) and return if @tax_returns.empty? || !@client.online_ctc?
```

Comment:
maybe instead of checking anything about CTC it should just look for efile submissions

that's the criteria that the navigation partial uses when choosing whether to hide/show the tab anyway
https://github.com/codeforamerica/vita-min/blob/main/app/views/hub/clients/_navigation.html.erb#L6